docs/tutorials/embedding/AdjacencySpectralEmbed.ipynb

```diff
@@ -46,14 +46,14 @@
     "undirected_sbm = sbm(2 * [n_verts], P)\n",
     "heatmap(undirected_sbm, title='2-block SBM (undirected)', inner_hier_labels=labels_sbm)\n",
     "directed_sbm = sbm(2 * [n_verts], P, directed=True)\n",
-    "heatmap(directed_sbm, title='2-block SBM (directed)', inner_hier_labels=labels_sbm)"
+    "heatmap(directed_sbm, title='2-block SBM (directed)', inner_hier_labels=labels_sbm);"
```

Comment:
Out of curiousity, why all the semicolons?

I like my semicolons as much as the next Java developer, but they're not exactly pythonic

Reply:
The semicolons on the last line of a code block suppress the output message from a graph in Jupyter.  Ex. in this notebook, without the semicolon, 
``<AxesSubplot:title={'center':'2-block SBM (directed)'}>``
gets printed above the heatmap graph.

Reply:
an alternative way to accomplish the same thing is to assign the result to an underscore ex.
``_ = heatmap(directed_sbm, title='2-block SBM (directed)', inner_hier_labels=labels_sbm)``

Reply:
Oh, okay - I've always done the `_ =`, but it's good to know that ending with `;` does it too.  Thanks!

Reply:
Actually, apparently netlify doesn't support the ``;`` method of suppressing graph output messages (even though jupyter notebook does).  So I have changed all the relevant lines to ``_ = `` instead.  Sorry for any confusion.